src/main/java/at/researchstudio/sat/merkmalservice/model/Project.java

```diff
@@ -0,0 +1,46 @@
+package at.researchstudio.sat.merkmalservice.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Project {
+    private final String name;
```

Comment:
description should also be extracted (imho)

Reply:
done